packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barrier-selector.jsx

```diff
@@ -47,6 +48,25 @@ const BarrierSelector = ({ barrier_1, onChange, setHoveredBarrier, turbos_barrie
 
     return (
         <React.Fragment>
+            {isMobile() ? (
+                <div className='mobile-widget'>
```

Comment:
These `div` with styles were taken from existing component (`MobileWidget` for stake and amount). In next card (add expanded barriers for mobile) I planned to apply existing `toggleBarriersTable` function on it.